packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1374,11 +1374,11 @@ - (NSDictionary *)multiFactorResolverToDict:(FIRMultiFactorResolver *)resolver {
 }
 
 - (NSString *)getJSFactorId:(NSString *)factorId {
-  if ([factorId isEqualToString:@"1"]) {
+  if (factorId == nil || [factorId isEqualToString:@"1"]) {
```

Comment:
If as I propose below we guard against nil ever coming in here then this change should be unnecessary.
I'm also not sure converting a "nil" into a return value that semantically means "phone is enrolled" is correct so this change is doubtful for me from a correctness perspective